std_msgs/Header header
ProcessedObject[] objects
